docs/process/releasing-updates.md

```diff
@@ -6,25 +6,60 @@ We have three channels to which we can release: `production`, `beta`, and `test`
 
 - `production` is the channel from which the general public downloads and receives updates. It should be stable and polished.
 
-- `beta` is released more often than `production`. It may be buggy and unpolished.
+- `beta` is released more often than `production`. It may be buggy and unpolished. For example: we should be able to release at any point from the master branch.
 
 - `test` is unlike the other two. It does not receive updates. Each test release is locked in time. It's used entirely for providing test releases.
 
 ## The Process
 
-From a clean working directory, set the `GITHUB_ACCESS_TOKEN` environment variable to a valid [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) and run:
+For releasing to `test`, skip to the `Releasing` header (no draft release needed).
+
+### 1. GitHub Access Token
+
+From a clean working directory, set the `GITHUB_ACCESS_TOKEN` environment variable to a valid [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) 
+
+You can check if it is set on MacOS by:
+```shellsession
+$ env GITHUB_ACCESS_TOKEN
+```
+
+You can check if it is set on Windows by:
+```shellsession
+$ echo %GITHUB_ACCESS_TOKEN%
+```
+
+If you are creating a new Personal Access Token on GitHub:
+* a reasonable description is: `Desktop Draft Release and Changelog Generator`
+* ONLY check the `read:org` checkbox for scopes/permissions
+
+You can add an access token as an environment variable on MacOS by: 
+```shellsession
+$ export GITHUB_ACCESS_TOKEN={your token here}
+```
+
+You can add an access token as an environment variable on Windows by: 
+```shellsession
+$ set GITHUB_ACCESS_TOKEN={your token here}
+```
+
+### 2. Create Draft Release
+
+Once our access token is all set, we run a script that works out the next version from what was previously published, based on the channel.
```

Comment:
:lipstick: Once our **personal** access token **is set**, **run the script** ... based on the **desired** channel